service_container.rst

```diff
@@ -690,6 +690,13 @@ service whose id is ``monolog.logger.request``.
     the *service* whose id is ``monolog.logger.request``, and not just the *string*
     ``monolog.logger.request``.
 
+.. _services-binding:
+
+Binding Arguments by Name or Type
+---------------------------------
+
+You can also use the ``bind`` keyword to bind specific arguments by name or type.
+
```

Comment:
This is a poor bootstrap of the `bind` section, just so I can link to it. i've commented in the issue about that feature (#8276) to note that this was created.